tasks/pipeline-dbconf.yml

```diff
@@ -40,17 +40,17 @@
     priv: "MCP.*:ALL"
     state: "present"
 
-- name: "Initialize database MCP"
+- name: "Load MySQL seed file"
   mysql_db:
     name: "MCP"
     state: "import"
     target: "{{ archivematica_src_dir }}/archivematica/src/MCPServer/share/mysql"
-  when: "createdb_result|changed"
+  when: "createdb_result|changed and archivematica_src_am_pre_migration|bool"
 
 # TODO: Does this need to run as root or user archivematica ??
-- name: "Invoke django syncdb"
+- name: "Run migrations"
   django_manage:
-    command: "syncdb"
+    command: "migrate"
```

Comment:
Will this break when running the playbook to install old branches (stable/1.4.x, etc that are on Django 1.5)? I think we need to have two separate tasks: one to run syncdb when archivematica_src_am_pre_migration=true and one to run migrate when archivematica_src_am_pre_migration=false.